plugins/out_opensearch/opensearch.c

```diff
@@ -875,6 +1059,18 @@ static void cb_opensearch_flush(struct flb_event_chunk *event_chunk,
         FLB_OUTPUT_RETURN(FLB_RETRY);
     }
 
+	/*
+	 * create index template if data stream mode is enabled
+	*/
+
+	if (ctx->data_stream_mode == FLB_TRUE) {
+		res = create_index_template(ctx);
+		flb_plg_debug(ctx->ins, "create index template=%i", res);
+		if (res != 0) {
+			FLB_OUTPUT_RETURN(FLB_RETRY);
+		}
+	}
+
```

Comment:
8 spaces for indentation is not correct for our coding style. Could you use 4 spaces indentations?